apps/bfd-server/bfd-server-war/src/test/resources/openapi/posts.yaml

```diff
@@ -206,7 +198,8 @@ paths:
                     (mbi, hicn, etc); the following are all valid values for Identifier, and all might represent the same resource:
                       - `identifier=https://bluebutton.cms.gov/resources/identifier/hicn-hash|<your hicn hash>`
                       - `identifier=https://bluebutton.cms.gov/resources/identifier/mbi-hash|<your mbi hash>`
-                  example: "https://bluebutton.cms.gov/resources/identifier/mbi-hash|<your mbi hash>"
+                      - `identifier=http://hl7.org/fhir/sid/us-mbi|<your mbi>`
+                  example: "http://hl7.org/fhir/sid/us-mbi|<your mbi>"
```

Comment:
I switched the example here to show the unhashed MBI since my understanding is we're encouraging consumers to use this endpoint with the explicit recommendation to use it without hashing.